src/compiler/parser.ts

```diff
@@ -2698,7 +2699,7 @@ namespace ts {
             }
         }
 
-        function isStartOfType(): boolean {
+        function isStartOfType(disableLookahead?: boolean): boolean {

```

Comment:
can we have a more declarative name, like inJSDocType or something?

Reply:
sure. done.